src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example1.java

```diff
@@ -0,0 +1,40 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.naming.MemberNameCheck"/>
+  <module name="com.puppycrawl.tools.checkstyle.checks.naming.ConstantNameCheck"/>
```

Comment:
What forcing you to define fully qualified name?

Reply:
@relentless-pursuit , please reply

Reply:
Sure @romani . So, it was more of a trial and error process. Initially, when i started working on this issue, i was constantly getting failure on the lines config comment not specific properly. And, was asking for a fully qualified name of the modules. However, now, i can see it doing fine without fully qualified name. I will try to push again with just module names.

Reply:
I started with what was given in the text file i.e. `<module name="MemberName"/>`. And, then i moved on to debugg the code. And, i assumed we need to use fully qualified names if we have more than one module. And, went with fully qualified name. Now, when i use <module name="MemberNameCheck"/>, it works fine.

Reply:
My bad. I was testing on a different test. Here, is the error when i don't use fully qualified name. 
com.puppycrawl.tools.checkstyle.api.CheckstyleException: Suppose, if my config looks like the below:
```
/*xml
<module name="Checker">
  <module name="TreeWalker">
  <module name="MemberName"/>
  <module name="com.puppycrawl.tools.checkstyle.checks.naming.ConstantNameCheck"/>
  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck">
    <property name="id" value="ParamNumberId"/>
  </module>
  <module name=
    "com.puppycrawl.tools.checkstyle.checks.whitespace.NoWhitespaceAfterCheck"/>
  <module name="com.puppycrawl.tools.checkstyle.checks.SuppressWarningsHolder"/>
  </module>
  <module name="com.puppycrawl.tools.checkstyle.filters.SuppressWarningsFilter"/>
</module>
*/
```
I am getting this error on running the tests. 
```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: Config comment not specified properly in /Users/piyushkumarsadangi/Development/new/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example1.java

...

Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to resolve module name: MemberName. Please check for spelling errors or specify fully qualified class name.
```

Reply:
To my understanding by using some logs in `getFullyQualifiedClassName`
```
input file path +++--->/Users/piyushkumarsadangi/Development/new/checkstyle/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example1.java
module name +++--->MemberName
formatted path +++--->.Users.piyushkumarsadangi.Development.new.checkstyle.src.xdocs-examples.resources.com.puppycrawl.tools.checkstyle.checks.annotation.suppresswarningsholder.Example1.java
endIndex +++--->-1
```

Reply:
Please debug why it happens
Here it works https://checkstyle.org/checks/naming/membername.html#Example1-config

Reply:
And, If there are any other functional Examples where we use multiple module in the config, please let me know. I can get some more idea from it. I don't remember working on anything like these, having multiple modules in a single config.

Reply:
> Please debug why it happens Here it works https://checkstyle.org/checks/naming/membername.html#Example1-config

Sure, will share any new insights if i get from it.

Reply:
It might be that our abstract test suit needs to be extended to support this. Please suggest solution.
If easy to fix, we can update in this PR, if not easy, we can keep fully qualified names and create separate issue on this.

Reply:
> It might be that our abstract test suit needs to be extended to support this. Please suggest solution. If easy to fix, we can update in this PR, if not easy, we can keep fully qualified names and create separate issue on this.

I have tried to provide a fix for the issue discussed here along with issue i opened few days back in #13845. Some checks are failing but test cases are running fine. Please let me know your feedback on the same. I would consider the above solution pushed in my recent patch just a hack rather than a sustainable solution.

Reply:
Ok, I see a problem, let resolve problems step by step.

Please do hardcoded of mapping of names to fully qualified names that requires only for this PR, put a comment that hack will stay until #13845 .
It will let us merge examples sooner, and work on smaller update to remove hack without big update for examples.

Reply:
> it can't find `MemberName` in the path of suppresswarningsholder. Therefore, expects a fully qualified name which is a provision in the code.

Sure, just to confirm, the observation i had over here, is it right? Or, is there something else i have missed out? And, will this issue be the same for config with multiple modules belonging to different packages?

Reply:
Yes, it will affect filters a lot , same problems will be as in this PR.
But stupid harcode will unblock examples update, and I hope someone will handle https://github.com/checkstyle/checkstyle/issues/13845 to resolve a problem forever.